### ISSUE TITLE
Making Search Box Corners Square

### ISSUE THREAD

**Issue by Creatoon** [CircuitVerse/CircuitVerse#2710]
**Describe the bug**
If we match the input boxes on the simulator page, we will find the corners of Search Box in the Circuit Elements Box and Properties box do not match, they all have to maintain same kind of corners.

**To Reproduce**
1. Go to https://circuitverse.org/simulator
2. Check the corners of Input boxes in both Circuit Elements Box and Properties Box

Before:-

![Screenshot 2021-12-13 at 23-23-08 CircuitVerse - Digital Circuit Simulator online](https://user-images.githubusercontent.com/65965202/145864028-ccbc280a-0fa5-47af-886b-d0667a6c5407.png) ![Screenshot 2021-12-13 at 23-23-24 CircuitVerse - Digital Circuit Simulator online](https://user-images.githubusercontent.com/65965202/145864039-e4f27ecf-8766-4217-b16e-46e072344bc2.png)

After:-
![Screenshot 2021-12-13 at 23-30-37 CircuitVerse - Digital Circuit Simulator online](https://user-images.githubusercontent.com/65965202/145864365-91f44088-927b-432e-8ac3-cc06a55802d9.png)

**Are you working on this issue? (Yes/No)**
Yes, If the issue is approved, I will make a pr.

**Comment by vedant-jain03** [comment]
I think this is the expected behavior as the search input box is used to get the input and search the results and the Properties box is used to input the prop. for respective circuit elements. There must be a difference between them.
Let's wait for @nitin10s assistance.

**Comment by nitin10s** [comment]
No, actually this is a problem Vedant and should be fixed, good find. Please note that the input height in the "after" image is less compared to the other fields.

**Comment by Creatoon** [comment]
> No, actually this is a problem Vedant and should be fixed, good find. Please note that the input height in the "after" image is less compared to the other fields.

I think the height is same, the only issue is with border-radius, am I right ?

**Comment by Creatoon** [comment]
@vedant-jain03 Can I make the pr for the same?

**Comment by Kunal0220** [comment]
hey @Creatoon , I can resolve this issue , Can I start working ?

**Comment by Creatoon** [comment]
@Kunal0220 hey, I just resolved this and within few hours gonna create a pr, if today I will not make any pr you can surely go for it 😊

**Comment by Kunal0220** [comment]
@Creatoon Ok Cool , I will check it tomorrow once more , btw Happy New Year in Advance 🙌🏻
